### PR TITLE
update hprose to 1.6.6

### PR DESCRIPTION
**Pull request by Lao-liu**
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-php/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

-----

Formula (master) ✔ brew install ./php71-hprose.rb
==> Downloading https://pecl.php.net/get/hprose-1.6.6.tgz
######################################################################## 100.0%
==> /usr/local/opt/php71/bin/phpize
==> ./configure --prefix=/usr/local/Cellar/php71-hprose/1.6.6_2 --with-php-config=/usr/local/opt/php71/bi
==> make
==> Caveats
To finish installing hprose for PHP 7.1:
  * /usr/local/etc/php/7.1/conf.d/ext-hprose.ini was created,
    do not forget to remove it upon extension removal.
  * Validate installation via one of the following methods:
  *
  * Using PHP from a webserver:
  * - Restart your webserver.
  * - Write a PHP page that calls "phpinfo();"
  * - Load it in a browser and look for the info on the hprose module.
  * - If you see it, you have been successful!
  *
  * Using PHP from the command line:
  * - Run `php -i "(command-line 'phpinfo()')"`
  * - Look for the info on the hprose module.
  * - If you see it, you have been successful!
==> Summary
🍺  /usr/local/Cellar/php71-hprose/1.6.6_2: 3 files, 353.8KB, built in 21 seconds

### CODE REVIEW

File: Formula/php56-hprose.rb [hunk 1]
```diff
@@ -2,20 +2,13 @@
...
   init
-  desc "Hprose for PHP."
+  desc "High Performance Remote Object Service Engine"
   homepage "https://pecl.php.net/package/hprose"
-  url "https://pecl.php.net/get/hprose-1.6.5.tgz"
-  sha256 "a453053719dad6352d54543d708d8d09b62de28df7b29fd47b4b5a9fcf8025a7"
+  url "https://pecl.php.net/get/hprose-1.6.6.tgz"
+  sha256 "29292d9ba15c3f838622bbf8f608a0fb4fb6bba6019f6e6bffe1eedb572881b8"
   head "https://github.com/hprose/hprose-pecl.git"
   revision 2
```
Comment (SMillerDev) [inline Formula/php56-hprose.rb]: Since you're updating the version you can remove this line.

File: Formula/php70-hprose.rb [hunk 2]
```diff
@@ -2,20 +2,13 @@
...
   init
-  desc "Hprose for PHP."
+  desc "High Performance Remote Object Service Engine"
   homepage "https://pecl.php.net/package/hprose"
-  url "https://pecl.php.net/get/hprose-1.6.5.tgz"
-  sha256 "a453053719dad6352d54543d708d8d09b62de28df7b29fd47b4b5a9fcf8025a7"
+  url "https://pecl.php.net/get/hprose-1.6.6.tgz"
+  sha256 "29292d9ba15c3f838622bbf8f608a0fb4fb6bba6019f6e6bffe1eedb572881b8"
   head "https://github.com/hprose/hprose-pecl.git"
   revision 2
```
Comment (SMillerDev) [inline Formula/php70-hprose.rb]: Since you're updating the version you can remove this line.

File: Formula/php71-hprose.rb [hunk 3]
```diff
@@ -4,17 +4,10 @@ class Php71Hprose < AbstractPhp71Extension
...
+  sha256 "29292d9ba15c3f838622bbf8f608a0fb4fb6bba6019f6e6bffe1eedb572881b8"
   head "https://github.com/hprose/hprose-pecl.git"
-
-  bottle do
-    cellar :any_skip_relocation
-    sha256 "3e7c75c6fbe6c0661825588a5c04c63d845ecd182ea9127403a711f02f53d963" => :el_capitan
-    sha256 "c09a2c86c56b66e2fc773f42119fda318327de18ae056ca28ba9fc0a759d2c28" => :yosemite
-    sha256 "e5f00549be6e9b4f02efb029c2367603fed626c72e841276e4f0ba88773aec72" => :mavericks
-  end
+  revision 2
```
Comment (SMillerDev) [inline Formula/php71-hprose.rb]: Since you're updating the version you can remove this line.

File: Formula/php56-hprose.rb [hunk 4]
```diff
@@ -2,20 +2,13 @@
...
 
-  bottle do
```
Comment (SMillerDev) [inline Formula/php56-hprose.rb]: Please don't remove the bottle blocks, they're used to provide rebuild binaries

File: Formula/php70-hprose.rb [hunk 5]
```diff
@@ -2,20 +2,13 @@
...
 
-  bottle do
```
Comment (SMillerDev) [inline Formula/php70-hprose.rb]: Please don't remove the bottle blocks, they're used to provide rebuild binaries

File: Formula/php71-hprose.rb [hunk 6]
```diff
@@ -4,17 +4,10 @@ class Php71Hprose < AbstractPhp71Extension
...
   desc "High Performance Remote Object Service Engine"
   homepage "https://pecl.php.net/package/hprose"
-  url "https://pecl.php.net/get/hprose-1.6.5.tgz"
-  sha256 "a453053719dad6352d54543d708d8d09b62de28df7b29fd47b4b5a9fcf8025a7"
-  revision 2
+  url "https://pecl.php.net/get/hprose-1.6.6.tgz"
+  sha256 "29292d9ba15c3f838622bbf8f608a0fb4fb6bba6019f6e6bffe1eedb572881b8"
   head "https://github.com/hprose/hprose-pecl.git"
-
-  bottle do
```
Comment (SMillerDev) [inline Formula/php71-hprose.rb]: Please don't remove the bottle blocks, they're used to provide rebuild binaries

File: Formula/php72-hprose.rb [hunk 7]
```diff
@@ -4,18 +4,10 @@ class Php72Hprose < AbstractPhp72Extension
   init
   desc "High Performance Remote Object Service Engine"
   homepage "https://pecl.php.net/package/hprose"
-  url "https://pecl.php.net/get/hprose-1.6.5.tgz"
-  sha256 "a453053719dad6352d54543d708d8d09b62de28df7b29fd47b4b5a9fcf8025a7"
+  url "https://pecl.php.net/get/hprose-1.6.6.tgz"
+  sha256 "29292d9ba15c3f838622bbf8f608a0fb4fb6bba6019f6e6bffe1eedb572881b8"
   head "https://github.com/hprose/hprose-pecl.git"
 
-  bottle do
```
Comment (SMillerDev) [inline Formula/php72-hprose.rb]: Please don't remove the bottle blocks, they're used to provide rebuild binaries

File: Formula/php72-hprose.rb [hunk 8]
```diff
@@ -4,18 +4,10 @@ class Php72Hprose < AbstractPhp72Extension
...
-    cellar :any_skip_relocation
-    rebuild 1
```
Comment (SMillerDev) [inline Formula/php72-hprose.rb]: Since you're updating the version you can remove this line.

File: Formula/php70-hprose.rb (hunk 2 above)
Comment (Lao-liu) [reply Formula/php70-hprose.rb]: @SMillerDev OK

File: Formula/php56-hprose.rb [hunk 9]
```diff
@@ -2,10 +2,9 @@
...
 class Php56Hprose < AbstractPhp56Extension
   init
-  desc "Hprose for PHP."
+  desc "High Performance Remote Object Service Engine"
   homepage "https://pecl.php.net/package/hprose"
-  url "https://pecl.php.net/get/hprose-1.6.5.tgz"
-  sha256 "a453053719dad6352d54543d708d8d09b62de28df7b29fd47b4b5a9fcf8025a7"
+  url "https://pecl.php.net/get/hprose-1.6.6.tgz"
   head "https://github.com/hprose/hprose-pecl.git"
   revision 2
```
Comment (SMillerDev) [inline Formula/php56-hprose.rb]: This line is no longer needed as you're updating the version

File: Formula/php70-hprose.rb [hunk 10]
```diff
@@ -2,10 +2,9 @@
...
 class Php70Hprose < AbstractPhp70Extension
   init
-  desc "Hprose for PHP."
+  desc "High Performance Remote Object Service Engine"
   homepage "https://pecl.php.net/package/hprose"
-  url "https://pecl.php.net/get/hprose-1.6.5.tgz"
-  sha256 "a453053719dad6352d54543d708d8d09b62de28df7b29fd47b4b5a9fcf8025a7"
+  url "https://pecl.php.net/get/hprose-1.6.6.tgz"
   head "https://github.com/hprose/hprose-pecl.git"
   revision 2
```
Comment (SMillerDev) [inline Formula/php70-hprose.rb]: This line is no longer needed as you're updating the version

File: Formula/php71-hprose.rb [hunk 11]
```diff
@@ -4,10 +4,9 @@ class Php71Hprose < AbstractPhp71Extension
   init
   desc "High Performance Remote Object Service Engine"
   homepage "https://pecl.php.net/package/hprose"
-  url "https://pecl.php.net/get/hprose-1.6.5.tgz"
-  sha256 "a453053719dad6352d54543d708d8d09b62de28df7b29fd47b4b5a9fcf8025a7"
-  revision 2
+  url "https://pecl.php.net/get/hprose-1.6.6.tgz"
   head "https://github.com/hprose/hprose-pecl.git"
+  revision 2
```
Comment (SMillerDev) [inline Formula/php71-hprose.rb]: This line is no longer needed as you're updating the version

File: Formula/php53-hprose.rb [hunk 12]
```diff
@@ -2,10 +2,9 @@
 
 class Php53Hprose < AbstractPhp53Extension
   init
-  desc "Hprose for PHP."
+  desc "High Performance Remote Object Service Engine"
   homepage "https://pecl.php.net/package/hprose"
-  url "https://pecl.php.net/get/hprose-1.6.5.tgz"
-  sha256 "a453053719dad6352d54543d708d8d09b62de28df7b29fd47b4b5a9fcf8025a7"
```
Comment (SMillerDev) [inline Formula/php53-hprose.rb]: We need this shasum to verify the downloaded file

File: Formula/php54-hprose.rb [hunk 13]
```diff
@@ -2,10 +2,9 @@
 
 class Php54Hprose < AbstractPhp54Extension
   init
-  desc "Hprose for PHP."
+  desc "High Performance Remote Object Service Engine"
   homepage "https://pecl.php.net/package/hprose"
-  url "https://pecl.php.net/get/hprose-1.6.5.tgz"
-  sha256 "a453053719dad6352d54543d708d8d09b62de28df7b29fd47b4b5a9fcf8025a7"
```
Comment (SMillerDev) [inline Formula/php54-hprose.rb]: We need this shasum to verify the downloaded file

File: Formula/php55-hprose.rb [hunk 14]
```diff
@@ -2,10 +2,9 @@
 
 class Php55Hprose < AbstractPhp55Extension
   init
-  desc "Hprose for PHP."
+  desc "High Performance Remote Object Service Engine"
   homepage "https://pecl.php.net/package/hprose"
-  url "https://pecl.php.net/get/hprose-1.6.5.tgz"
-  sha256 "a453053719dad6352d54543d708d8d09b62de28df7b29fd47b4b5a9fcf8025a7"
```
Comment (SMillerDev) [inline Formula/php55-hprose.rb]: We need this shasum to verify the downloaded file

File: Formula/php56-hprose.rb [hunk 15]
```diff
@@ -2,10 +2,9 @@
 
 class Php56Hprose < AbstractPhp56Extension
   init
-  desc "Hprose for PHP."
+  desc "High Performance Remote Object Service Engine"
   homepage "https://pecl.php.net/package/hprose"
-  url "https://pecl.php.net/get/hprose-1.6.5.tgz"
-  sha256 "a453053719dad6352d54543d708d8d09b62de28df7b29fd47b4b5a9fcf8025a7"
```
Comment (SMillerDev) [inline Formula/php56-hprose.rb]: We need this shasum to verify the downloaded file

File: Formula/php70-hprose.rb [hunk 16]
```diff
@@ -2,10 +2,9 @@
 
 class Php70Hprose < AbstractPhp70Extension
   init
-  desc "Hprose for PHP."
+  desc "High Performance Remote Object Service Engine"
   homepage "https://pecl.php.net/package/hprose"
-  url "https://pecl.php.net/get/hprose-1.6.5.tgz"
-  sha256 "a453053719dad6352d54543d708d8d09b62de28df7b29fd47b4b5a9fcf8025a7"
```
Comment (SMillerDev) [inline Formula/php70-hprose.rb]: We need this shasum to verify the downloaded file

File: Formula/php71-hprose.rb [hunk 17]
```diff
@@ -4,10 +4,9 @@ class Php71Hprose < AbstractPhp71Extension
   init
   desc "High Performance Remote Object Service Engine"
   homepage "https://pecl.php.net/package/hprose"
-  url "https://pecl.php.net/get/hprose-1.6.5.tgz"
-  sha256 "a453053719dad6352d54543d708d8d09b62de28df7b29fd47b4b5a9fcf8025a7"
```
Comment (SMillerDev) [inline Formula/php71-hprose.rb]: We need this shasum to verify the downloaded file

File: Formula/php72-hprose.rb [hunk 18]
```diff
@@ -4,8 +4,7 @@ class Php72Hprose < AbstractPhp72Extension
   init
   desc "High Performance Remote Object Service Engine"
   homepage "https://pecl.php.net/package/hprose"
-  url "https://pecl.php.net/get/hprose-1.6.5.tgz"
-  sha256 "a453053719dad6352d54543d708d8d09b62de28df7b29fd47b4b5a9fcf8025a7"
```
Comment (SMillerDev) [inline Formula/php72-hprose.rb]: We need this shasum to verify the downloaded file

**Comment by Lao-liu** [comment]
@SMillerDev OK

File: Formula/php53-hprose.rb [hunk 19]
```diff
@@ -2,11 +2,11 @@
 
 class Php53Hprose < AbstractPhp53Extension
```
Comment (SMillerDev) [inline Formula/php53-hprose.rb]: Currently PHP 5.3 builds are broken, for now it's better to remove the formula update for 5.3 as it'll make the build fail.

Comment (SMillerDev) [reply Formula/php53-hprose.rb]: And it's a very old and outdated version of PHP that shouldn't be used anyways.

Comment (Lao-liu) [reply Formula/php53-hprose.rb]: @SMillerDev  OK

**Comment by SMillerDev** [comment]
Thanks @Lao-liu !

**Pull request closed without merging**